src/converters/fromZigbee.ts

```diff
@@ -3508,17 +3522,18 @@ export const enocean_ptm216z: Fz.Converter = {
     },
 };
 // biome-ignore lint/style/useNamingConvention: ignored using `--suppress`
-export const _8840100H_water_leak_alarm: Fz.Converter = {
+export const _8840100H_water_leak_alarm: Fz.Converter<"haApplianceEventsAlerts", undefined, "commandAlertsNotification"> = {
     cluster: "haApplianceEventsAlerts",
     type: "commandAlertsNotification",
     convert: (model, msg, publish, options, meta) => {
         const alertStatus = msg.data.aalert;
         return {
+            // @ts-expect-error TODO: wrong type assumed
```

Comment:
@presslab-us since you introduced the Waxxman 8840100H in https://github.com/Koenkk/zigbee-herdsman-converters/pull/1201, can you confirm it correctly detects the water leak? It seems this converter code is wrong and I'm worried that it always returns `water_leak: false` (even when there is one)

Reply:
Disabled this device in https://github.com/Koenkk/zigbee-herdsman-converters/pull/9915